README.md

```diff
@@ -30,13 +22,9 @@ Please see the [help page](https://cloud.google.com/dotnet/docs/reference/help/r
 
 ## Supported Frameworks
 
-(Note: this set may change in early 2024, as per
-[this announcement
-issue](https://github.com/googleapis/google-api-dotnet-client/issues/2561).
-
-* .NET Framework 4.5+
-* netstandard1.3, providing .NET Core support
-* netstandard2.0, providing [ASP.NET Core Support](https://www.nuget.org/packages/Google.Apis.Auth.AspNetCore)
+* .NET Framework 4.6.2+
+* netstandard2.0, providing [ASP.NET 3 Core Support](https://www.nuget.org/packages/Google.Apis.Auth.AspNetCore3)
```

Comment:
I'd personally ditch the "providing ASP.NET 3 Core support" part. I think we'd support netstandard2.0 anyway - it's a fairly common target for other libraries (which might want to depend on us).

Reply:
Yep, I didn't like that either very much. Done.
And also changed to .NET Standard 2.0